src/i18n/Translate.js

```diff
@@ -1,9 +1,15 @@
 import { PropTypes } from 'react';
 import { getContext } from 'recompose';
 
-const translate = getContext({
-    translate: PropTypes.func.isRequired,
-    locale: PropTypes.string.isRequired,
-});
+const translate = (BaseComponent) => {
```

Comment:
No: we need it otherwise the component don't transfer the `defaultProps`, hence causing some issues with the `input is not defined` due to the `addField` non-transfer.